pallets/gear/src/lib.rs

```diff
@@ -685,7 +637,7 @@ pub mod pallet {
         pub fn process_queue() -> Weight {
             let mut ext_manager = ExtManager::<T>::default();
 
-            let weight = Self::gas_allowance() as Weight;
+            let weight = GasPallet::<T>::gas_allowance() as Weight;
```

Comment:
I would like this small refactoring to be in a separate PR

Reply:
Yeah, this one is really small. Don't want to waste time for it by creating separate PR. This changes seems reasonable and balanced between other changes. How do you feel?